lib/omnibus/software.rb

```diff
@@ -667,9 +668,7 @@ def with_standard_compiler_flags(env = {}, opts = {})
             # TODO: This was true of our old TDM gcc 4.7 compilers. Is it still
             # true with mingw-w64?
             #
-            # XXX: Temporarily turning -O3 into -O2 -fno-lto to work around some
-            # weird linker issues.
-            "CFLAGS" => "-I#{install_dir}/embedded/include #{arch_flag} -O2 -fno-lto #{opt_flag}",
+            "CFLAGS" => "-I#{install_dir}/embedded/include #{arch_flag} -O3 #{opt_flag}",
```

Comment:
Good to have:  the original fuckery was speculative anyway.  This just puts us back into O3.